OpenRA.Game/Graphics/SpriteCache.cs

```diff
@@ -82,7 +87,7 @@ public void LoadReservations(ModData modData)
 				string Filename,
 				int FrameIndex,
 				bool Premultiplied,
-				Func<ISpriteFrame, ISpriteFrame> AdjustFrame,
+				Func<ISpriteFrame, int, int, ISpriteFrame> AdjustFrame,
```

Comment:
I think introducing a custom delegate could be nice here, e.g.
`public delegate ISpriteFrame AdjustFrame(ISpriteFrame input, int index, int total);`

That would allow for some documentation as well as avoiding having to change the signature in so many places in the future should we ever tweak it again.